src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class RightTriangle  extends Figure implements AreaCalculator{
+
+    private int firstLag;
+
+    public RightTriangle(String color, String name, int firstLag) {
+        super(color, name);
+        this.firstLag = firstLag;
+    }
+
+    @Override
+    public int calculateArea() {
+        return (int)(Math.pow(firstLag, 2)*(Math.sqrt(3))/4);
```

Comment:
The formula used for calculating the area is incorrect for a right triangle. The area of a right triangle should be calculated as `(base * height) / 2`. If `firstLag` represents one of the legs, ensure the calculation reflects the correct formula.